concent_api/core/transfer_operations.py

```diff
@@ -39,15 +39,14 @@ def verify_file_status(
     """
 
     encoded_client_public_key = b64encode(client_public_key)
-    force_get_task_result_list = Subtask.objects.filter(
+    subtasks_list = Subtask.objects.select_for_update().filter(
         requestor__public_key  =encoded_client_public_key,
         state                  = Subtask.SubtaskState.FORCING_RESULT_TRANSFER.name,  # pylint: disable=no-member
     )
 
-    for get_task_result in force_get_task_result_list:
-        report_computed_task    = deserialize_message(get_task_result.report_computed_task.data.tobytes())
+    for subtask in subtasks_list:
+        report_computed_task    = deserialize_message(subtask.report_computed_task.data.tobytes())
```

Comment:
Please remove the whitespace here according to our new style.

Reply:
done